pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -23,8 +23,20 @@ type HyperConvergedSpec struct {
 
 	// LocalStorageClassName the name of the local storage class.
 	LocalStorageClassName string `json:"LocalStorageClassName,omitempty"`
+
+	// Version is the HCO version
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
If the version is not configurable...why would we add it to the spec?

Reply:
Not sure I follow - currently, the UI takes it from the spec. Is there anyway else to set it?

Reply:
We aren't allowing configuration of this value. So it should not be on the spec.

We could create an issue against the UI to make it so they could read it off the status.